src/actions/cra.actions.js

```diff
@@ -248,3 +252,43 @@ function countByPermanence(permanenceId) {
     return { type: 'COUNT_CRA_PERMANENCE_FAILURE', error };
   }
 }
+
+function searchSuggestion(theme, sousTheme) {
+  return dispatch => {
+    dispatch(request(sousTheme));
+    craService.searchSuggestion(theme, sousTheme)
+    .then(
+      result => {
+        dispatch(success(result.sousThemes));
+      },
+      error => {
+        dispatch(failure(error));
+      }
+    );
+  };
+
+  function request(suggestion) {
+    return { type: 'SEARCH_SOUS_THEMES_CRA_REQUEST', suggestion };
+  }
```

Comment:
Pas utilisé dans le reducer
```suggestion
  function request() {
    return { type: 'SEARCH_SOUS_THEMES_CRA_REQUEST'};
  }
```